### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ixboy**
Hello this is my React/Redux Final Capstone project, bellow are the learning objectives of this project:

    Use React documentation.
    Use React components.
    Use React props.
    Use React Router.
    Connect React and Redux.
    Handle events in a React app.
    Write integration tests with a React testing library.
    Use styles in a React app.
    Use React life cycle methods.
    Apply React best practices and language style guides in code.
    Use store, actions and reducers in React.

Please Review my project and let me know if there's any changes I should make.
Thank You.
Link to the video presenting the project:  [CLICK HERE](https://www.loom.com/share/bae2515925ff4e06a99413534a686ec9).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,65 @@
...
+  <a href="https://github.com/ixboy/countries-covid19-tracker">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/ixboy/countries-covid19-tracker?color=pink&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/ixboy">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/ixboy?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid-19 World Data Tracker
```
Comment (TedLivist) [inline README.md:13]: - [ ] In keeping with the Creative Commons License, in your README kindly add acknowledgment for the design author, Nelson Sakwa.

File: src/components/ListCountries/ListCountries.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/no-array-index-key */
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import CountUp from 'react-countup';
+import { NavLink } from 'react-router-dom';
+import { fetchCountries } from '../../redux/worldCountries/worldCountries';
+import styles from './ListCountries.module.css';
+
+const ListCountries = () => {
```
Comment (TedLivist) [inline src/components/ListCountries/ListCountries.js:10]: - [ ] Your live demo is missing the search functionality you have implemented. Kindly ensure this is rectified.

  ismail-search-live

File: src/components/CountryDetails/CountryDetails.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { fetchCountries } from '../../redux/worldCountries/worldCountries';
+import Cards from './Cards';
+import styles from './CountryDetails.module.css';
+
+const CountryDetails = () => {
```
Comment (TedLivist) [inline src/components/CountryDetails/CountryDetails.js:8]: ### Ensure that these functionalities reflect in your live demo as well
<hr />

- [ ] You app is missing the header as contained in the design. Kindly add it.

  ixboy-navbar
 
- [ ] When a user navigates to the **details** page on the top of the page, on the header above, there should be a `<` (**back**) button that navigates back to the Home page when clicks. Kindly add this functionality

  ixboy-back

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ixboy,

Good job so far!
✔️ Good documentation
✔️ Implementation of search functionality
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ixboy) [reply README.md:13]: @TedLivist  Thank You for your reviews I really appreciate. However I am pretty sure my design is way too different from Nelson Sakwa's designe, I don't see why I should give him credits if I did not copy from his design.
Anyways I just gave him credits just for the sake of getting my project approved...

File: README.md [hunk 4]
```diff
@@ -1,70 +1,73 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `yarn build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./MIT.md) licensed.
```
Comment (Rustamxon7) [inline README.md:73]: - [ ] [OPTIONAL] Please do not forget to add MIT License before merging the project!

**Review by Rustamxon7 — APPROVED** [review]
## Hello @ixboy 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**